ui-particles-angular/projects/ui-policy-studio-angular/src/lib/gio-policy-studio.apim.stories.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { Meta, moduleMetadata } from '@storybook/angular';
+import { action } from '@storybook/addon-actions';
+import { Story } from '@storybook/angular/dist/ts3.9/client/preview/types-7-0';
+import { GioPolicyStudioComponent } from './gio-policy-studio.component';
+import { GioPolicyStudioModule } from './gio-policy-studio.module';
+
+import { apimPolicies, apimResourceTypes, apimDefinition, apimFlow, apimPropertyProviders } from '../testing';
+
+export default {
+  title: 'Policy Studio / APIM',
+  component: GioPolicyStudioComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [GioPolicyStudioModule.forSpecs()],
+    }),
+  ],
+} as Meta;
+
+export const Default: Story = {
+  render: props => ({
+    template: `<gio-policy-studio
+    [canAdd]='canAdd'
+    [canDebug]='canDebug'
+    [hasResources]='hasResources'
+    [hasPolicyFilter]='hasPolicyFilter'
+    [hasProperties]='hasProperties'
+    [sortable]='sortable'
+    [policies]='policies'
+    [definition]='definition'
+    [services]='services'
+    [flowSchema]='flowSchema'
+    [resourceTypes]='resourceTypes'
+    [flowsTitle]='flowsTitle'
+    [propertyProviders]='propertyProviders'
+    [readonlyPlans]='readonlyPlans'
+    [dynamicPropertySchema]='dynamicPropertySchema'
+    [debugResponse]='debugResponse'
+    [configurationInformation]='configurationInformation'
+    (save)="saveAll($event)"
+    >
```

Comment:
maybe you can add [hasConditionalSteps] ?

Reply:
for the moment I just move the code of apim here.

In the next PR I think to make a story by apim use case : organization settings vs api
So it will happen with